crates/lib/kajiya-backend/src/vulkan/device.rs

```diff
@@ -327,18 +331,21 @@ impl Device {
 
                 assert!(shader_float16_int8.shader_int8 != 0);
 
-                assert!(vulkan_memory_model.vulkan_memory_model != 0);
-
                 #[cfg(feature = "ray-tracing")]
                 {
+                    assert!(descriptor_indexing.shader_uniform_buffer_array_non_uniform_indexing != 0);
+                    assert!(descriptor_indexing.shader_storage_buffer_array_non_uniform_indexing != 0);
+
+                    assert!(vulkan_memory_model.vulkan_memory_model != 0);
+
```

Comment:
macOS doesn't support these features, but I don't think that they're used without ray tracing anyway.